integration-tests/codestarts/src/test/java/com/vaadin/flow/quarkus/it/CodestartTestUtils.java

```diff
@@ -0,0 +1,58 @@
+package com.vaadin.flow.quarkus.it;
```

Comment:
This is for all classes. Not sure if we shall add the license headers everywhere. I observed in multiple classes it is added, even in IntegrationTests like [`RouteContextIT`](https://github.com/vaadin/quarkus/blob/master/integration-tests/common-test-code/src/test/java/com/vaadin/flow/quarkus/it/RouteContextIT.java#L1)

```
/*
 * Copyright 2000-2021 Vaadin Ltd.
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not
 * use this file except in compliance with the License. You may obtain a copy of
 * the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 * License for the specific language governing permissions and limitations under
 * the License.
 */
 ```

Reply:
Yes, I agree, then let's do it in a separate PR to not make noise here! 👍

Reply:
Issue was created:
- https://github.com/vaadin/quarkus/issues/120